website/hw/hw8.rst

```diff
@@ -638,8 +768,8 @@ Now, from your pipeline’s page, do the following:
    of the page.
 -  Enable “Create new review apps for new pull requests automatically”
 -  Do NOT enable “Wait for CI to pass” (this refers to Heroku’s own CI,
-   and we are already using Travis for CI)
--  Enable “Destroy stale review apps automatically”
+   and we are already using GitHub Actions for CI)
+-  Enable “Destroy stale review apps automatically”.
```

Comment:
It lets you specify a number of days
(It says "After 5 days without any deploys", where "After 5 days" is a dropdown menu)

Reply:
Clarified that they can leave the default value selected.